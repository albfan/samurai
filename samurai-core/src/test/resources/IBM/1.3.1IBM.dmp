1XMTHDINFO Thread InfoNULL -----------NULL2XMFULLTHDDUMP Full thread dump Classic VM (J2RE 1.3.1 IBM Windows 32build cn131-20030711a, native threads):3XMTHREADINFO "Finalizer" (TID:0x101B4100, sys_thread_t:0x998610,state:CW, native ID:0x94) prio=84XESTACKTRACE at java.lang.Object.wait(Native Method)4XESTACKTRACE at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:133)4XESTACKTRACE at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:148)4XESTACKTRACE at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:213)3XMTHREADINFO "Reference Handler" (TID:0x101B4148,sys_thread_t:0x97FD48, state:CW, native ID:0x1E0) prio=104XESTACKTRACE at java.lang.Object.wait(Native Method)4XESTACKTRACE at java.lang.Object.wait(Object.java:429)4XESTACKTRACE at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:131)3XMTHREADINFO "Signal dispatcher" (TID:0x101B4190,sys_thread_t:0x97D230, state:R, native ID:0x608) prio=53XMTHREADINFO "main" (TID:0x101B41D8, sys_thread_t:0x3E510, state:R,native ID:0xA0) prio=54XESTACKTRACE at InfiniteLoop.hang2(InfiniteLoop.java:14)4XESTACKTRACE at InfiniteLoop.hang1(InfiniteLoop.java:10)4XESTACKTRACE at InfiniteLoop.main(InfiniteLoop.java:6)NULL------------------------------------------------------------------------