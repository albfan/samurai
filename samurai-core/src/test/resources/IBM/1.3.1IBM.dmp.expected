ftd.0.deadLockSize=0
ftd.0.threadCount=4
#"Finalizer" (TID:0x101B4100, sys_thread_t:0x998610,state:CW, native ID:0x94) prio=8
#at java.lang.Object.wait(Native Method)
#at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:133)
#at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:148)
#at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:213)
td.0.0.blockedObjectId=null
td.0.0.blockerId=null
td.0.0.condition=(TID:0x101B4100, sys_thread_t:0x998610,state:CW, native ID:0x94) prio=8
td.0.0.header="Finalizer" (TID:0x101B4100, sys_thread_t:0x998610,state:CW, native ID:0x94) prio=8
td.0.0.id=0x998610
td.0.0.name=Finalizer
td.0.0.isBlocked=false
td.0.0.isBlocking=false
td.0.0.isDaemon=false
td.0.0.isDeadLocked=false
td.0.0.isIdle=true

#"Reference Handler" (TID:0x101B4148,sys_thread_t:0x97FD48, state:CW, native ID:0x1E0) prio=10
#at java.lang.Object.wait(Native Method)
#at java.lang.Object.wait(Object.java:429)
#at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:131)
td.0.1.blockedObjectId=null
td.0.1.blockerId=null
td.0.1.condition=(TID:0x101B4148,sys_thread_t:0x97FD48, state:CW, native ID:0x1E0) prio=10
td.0.1.header="Reference Handler" (TID:0x101B4148,sys_thread_t:0x97FD48, state:CW, native ID:0x1E0) prio=10
td.0.1.id=0x97FD48
td.0.1.name=Reference Handler
td.0.1.isBlocked=false
td.0.1.isBlocking=false
td.0.1.isDaemon=false
td.0.1.isDeadLocked=false
td.0.1.isIdle=true

#"Signal dispatcher" (TID:0x101B4190,sys_thread_t:0x97D230, state:R, native ID:0x608) prio=5
td.0.2.blockedObjectId=null
td.0.2.blockerId=null
td.0.2.condition=(TID:0x101B4190,sys_thread_t:0x97D230, state:R, native ID:0x608) prio=5
td.0.2.header="Signal dispatcher" (TID:0x101B4190,sys_thread_t:0x97D230, state:R, native ID:0x608) prio=5
td.0.2.id=0x97D230
td.0.2.name=Signal dispatcher
td.0.2.isBlocked=false
td.0.2.isBlocking=false
td.0.2.isDaemon=false
td.0.2.isDeadLocked=false
td.0.2.isIdle=false

#"main" (TID:0x101B41D8, sys_thread_t:0x3E510, state:R,native ID:0xA0) prio=5
#at InfiniteLoop.hang2(InfiniteLoop.java:14)
#at InfiniteLoop.hang1(InfiniteLoop.java:10)
#at InfiniteLoop.main(InfiniteLoop.java:6)
td.0.3.blockedObjectId=null
td.0.3.blockerId=null
td.0.3.condition=(TID:0x101B41D8, sys_thread_t:0x3E510, state:R,native ID:0xA0) prio=5
td.0.3.header="main" (TID:0x101B41D8, sys_thread_t:0x3E510, state:R,native ID:0xA0) prio=5
td.0.3.id=0x3E510
td.0.3.name=main
td.0.3.isBlocked=false
td.0.3.isBlocking=false
td.0.3.isDaemon=false
td.0.3.isDeadLocked=false
td.0.3.isIdle=false

